easy-setup.sh

```diff
@@ -0,0 +1,82 @@
+OS_BASE="$(uname)"
+BUILD_MAC=$"Darwin"
+BUILD_LIN=$"Linux"
+BUILD_WIN="Windows"
+KERALARESCUE_REPO_LINK="https://github.com/IEEEKeralaSection/rescuekerala.git"
+
+# Commands for Apple Mac
+if [ $OS_BASE == $BUILD_MAC ] 
+then
+# Install Git and the repo for Kerala Rescue
+brew install git
+# ======= Uncomment the following line if you do not have the repository downloaded
+# git clone $KERALARESCUE_REPO_LINK
+echo "Repository is downloaded now you are ready to go"
+cd rescuekerala/
+# Install python and pip in mac
+brew install python
+# Install requirements.txt
+pip3 install -r requirements.txt
+pip3 install -r requirements_debug.txt
+# Implementing docker.sh
+chmod 777 docker.sh
+sh docker.sh
+# PostgreSQL : Expecting that brew is preinstalled in your mac
+brew install postgres
+ln -sfv /usr/local/opt/postgresql/*.plist ~/Library/LaunchAgents
```

Comment:
Is this not working? `brew services start postgres`?

Reply:
Did not work on my mac the first time, but the commands in this PR will work. Not the straightforward method, but it has a degree of certainty